src/themes/default/elements/icon.variables

```diff
@@ -455,7 +455,7 @@ Icons are order A-Z in their group, Solid, Outline, Thin (Pro only) and Brand
     folder_minus: \f65d;
     folder_open: \f07c;
     folder_plus: \f65e;
-    font: \f031;
+    font: \f031; // stylelint-disable-line font-family-name-quotes
```

Comment:
ok, we just need to keep in mind, in case we update the icons via https://github.com/fomantic/create-fomantic-icons 
where this hint would otherwise get lost again

Reply:
I figured out the solution is to quote the values - fixed in separated PR, see https://github.com/fomantic/Fomantic-UI/pull/2610/commits/76b17f7eb9f306d133e9cbd1209876884f6471ba#r1049050447